examples/quickstart/requirements.txt

```diff
@@ -1,4 +1,5 @@
 pipecat-ai[daily,silero,webrtc,websocket,cartesia,google]
```

Comment:
Do we want to not even include Daily? We don't mention it in the README at all...